Jobs/TELEPORT.eai

```diff
@@ -17,6 +17,7 @@ function TeleportJob takes unit hu returns nothing
     set teleporting = false
     return
   endif
+  call CaptainGoHome()
```

Comment:
This will happen post teleport, which would mean units hang around in combat instead of running away.